package.json

```diff
@@ -23,6 +23,7 @@
         "nyc": "^15.1.0",
         "rimraf": "^3.0.2",
         "sinon": "^9.0.3",
+        "sinon-chai": "^3.5.0",
```

Comment:
I prefered to use syntax in https://www.chaijs.com/plugins/sinon-chai/ for spies. So added this dev dependency